temporal-sdk/src/test/java/io/temporal/worker/WorkerStressTests.java

```diff
@@ -19,7 +19,7 @@
 
 package io.temporal.worker;
 
-import static io.temporal.workflow.WorkflowTest.NAMESPACE;
+import static io.temporal.testing.SDKTestWorkflowRule.NAMESPACE;
```

Comment:
I would make this constant private and use `testRule.getTestEnvironment().getNamespace()` instead

Also, consider adding the most used methods to the rule itself. For example `testRule.getNamespace()`, `testRule.newWorkflowStub()`, `testRule.currentTimeMillis()`, etc.

Reply:
NAMESPACE is used in shared/TestActivities.java for example. If we make it private, then how should it be accessed there?

Reply:
You can pass namespace to a TestActivityImpl constructor.